_content/av/reincarnation-in-tibetan-buddhism_gamble-ruth.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Reincarnation in Tibetan Buddhism: The Third Karmapa and the Invention of a Tradition"
+subcat: podcast
+authors:
+  - "Ruth Gamble"
+editor: "Connie Kassor"
+source_url: "https://newbooksnetwork.com/ruth-gamble-reincarnation-in-tibetan-buddhism-the-third-karmapa-and-the-invention-of-a-tradition-oxford-up-2018"
+drive_links:
+  - "https://drive.google.com/file/d/1XEhLjwoXKwU356Zz7pPJHoZk3GBfFKvg/view?usp=drive_link"
+course: tibetan 
+tags:
+  - rebirth
+year: 2018
+publisher: newbooksnetwork
+minutes: 42
+---
+
+"Reincarnation in Tibetan Buddhism" examines the life of the Third Karmapa, Rangjung Dorjé, and how his autobiographical writings helped shape the tulku tradition. In this interview, the author reflects on the broader historical and contemporary significance of his legacy, as well as how her personal experiences, have influenced her scholarly perspective.
```

Comment:
A quote from the clip and a description of the clip. We're just adding the clip not the episode.